.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
```

Comment:
The npm script to fix lint problems is `lint:fix`, not just `lint`. So, change this line to `npm run lint:fix`.